contrib/bdii/htcondor-ce-provider

```diff
@@ -224,6 +257,10 @@ def main():
             total_jobs = total_vo_jobs.get(vo, 0)
             idle_jobs = idle_vo_jobs.get(vo, 0)
             running_jobs = running_vo_jobs.get(vo, 0)
+            resource_keys = ""
+            for resource in resources:
+                resource_keys += "GLUE2ComputingShareExecutionEnvironmentForeignKey: %s_%s\n" % (central_manager, resource)
+                resource_keys += "GLUE2ShareResourceForeignKey: %s_%s\n" % (central_manager, resource)
```

Comment:
Other parts of this file use `format` since `%` is considered "old-style": https://docs.python.org/release/3.1.5/library/stdtypes.html#old-string-formatting-operations

But other inconsistencies already exist.